### DISCUSSION TITLE
Auto store and farms like the native's

### DISCUSSION THREAD

**Discussion by BraianCAESAR** [Keriew/augustus#264] [Ideas]
Is it possible to make resources from farms to getting automaticcally into the granary, without the men carrying it? AND 4 squares farms too, or 1 square farms, 2 squares, like native farms. So you can make small towns around the map. Small warehouses, etc.

**Comment by Areldir** [comment]
I don't expect we'll be making the 1st idea. The whole premise of Caesar 3 is its walker system, so removing a crucial aspect of the core gameplay is unlikely. 
Eventually, we will be looking at separating farms out into 2 component parts: the farmhouse and its surrounding fields (similar to the later Impressions game "Emperor"). The fields will 1x1 tiles, and players will be able to place them in a set radius around a farmhouse, allowing for a more natural and flexible system which accommodates rough/broken terrain.

**Reply by BraianCAESAR** [reply]
Thx Thx! This would be very cool! The cityes will be more beautifull too!

**Reply by crudelios** [reply]
Yes, we have been talking about changing farms for a future update. It won't be any time soon though 😉
